docs/tutorials/data_visualization.ipynb

```diff
@@ -241,36 +349,98 @@
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "titanic_numerical.plot_histograms()"
    ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "start_time": "2023-06-16T08:16:56.298442Z",
+     "end_time": "2023-06-16T08:16:58.271293Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's look at the different histograms of certain columns:\n",
+    "* According to the `age` histogram, a majority of the passengers were aged between 20 and 40. If the bars within this range are added up, about 750 passengers fall between this age range. Just a few passengers were above 75 years.\n",
+    "    * It is worth noting, that we can use the `normal distribution` here to approximate the data and future measurements. The normal distribution is a symmetric bell-shaped curve with a continuous range from -infinity to +infinity and has a common value for mean, median, and mode.\n",
+    "* The `siblings_spouses` histogram illustrates that a majority of the passengers, about 900 had no siblings nor spouses. Meanwhile, for the rest of the passengers, nearly every passenger had a sibling or spouse.\n",
+    "* As per the `parents_children` histogram, nearly every passenger was accompanied by a parent or child.\n",
+    "* In the histogram for the `travel_class` column, it can be observed that more than 50% of the passengers were categorised as 3rd class passengers. Just about half of the remaining passengers were categorised as 2nd class and the rest were 1st class passengers.\n",
+    "* The last histogram, `survived` depicts the survival rate of all passengers. About 800 passengers had a survival rate between 0.00 and 0.13 meanwhile more than 500 passengers had very high survival rates.\n"
```

Comment:
again `survived` is always either 0 or 1 so it isn't between 0 and 0.13. 
about 800 passengers have a 0, so they dies and about 500 have a 1 so they survived.